### ISSUE TITLE
NTR ATPase activity, coupled to translocation of iron-sulfur clusters

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#19963]
This could be related to #17289 

I'd like to request
ATPase activity, coupled to translocation of iron-sulfur clusters
def: Catalysis of the reaction: ATP + H2O = ADP + phosphate, to directly drive the translocation of iron-sulfur clusters from one protein to another

this is for a chaperone that catalyzes the transfer of iron-sulfur clusters from one protein to another (ssc1 in fungi (ssq1in cerevisiae), hsc70 in higher eukaryotes) (left hand mitochondrial side in schematic below)

![image](https://user-images.githubusercontent.com/12727506/92603098-d1687c00-f2a6-11ea-851c-300ad2e5fe85.png)

I have annotated the proteins to this EC (which I hope is correct)
EC 7.2.-.- - Catalysing the translocation of inorganic cations

**Comment by pgaudet** [comment]
I think this is what we have defined as a 'carrier activity', http://amigo.geneontology.org/amigo/term/GO:0140132#display-lineage-tab

GO:0140132 iron-sulfur cluster carrier activity Binding to an iron-sulfur cluster and delivering it to an acceptor molecule.

Does that cover it ? If you want I can add the synonym. 

Thanks, Pascale

**Comment by ValWood** [comment]
That seems correct to me, but it seems that a descendent is required to reflect the  ATPase catalytic activity in this instance?

**Comment by Antonialock** [comment]
I don't think the ATPase ever touches the iron-sulfur cluster. It has ATPase activity, and protein binding activity, and it seems to facilitate transfer of the cluster from one protein to another.

**Comment by pgaudet** [comment]
But the activity is ATP-dependent ? Sounds like the mechanism is different from that of NfuA of E. coli ? See PMID:22966982

**Comment by ValWood** [comment]
I think that is likely what the obsoleted yesterday term was for!

GO:0043492 ATPase activity, coupled to movement of substances

movement of substances = translocation
It's a translocator = movement of substances
but not a transmembrane transporter
and an ATPase

Although if so it should not be a parent of transmembrane transporter.

I think one of the problems is that people use the word translocator to refer to these, which we have called carriers, and to  some classes transmembrane transporters. It s would be useful to have  some expert input  on this issue, because we always come back to the same problem.

**Comment by ValWood** [comment]
I agree with Antonia that  this is an ATP driven process. These proteins are clearly ATPases
https://www.pombase.org/gene/SPAC664.11

Like all J-proteins, regardless of biological function, Jac1 has the fundamental function of stimulating the ATPase activity of its partner Hsp70, enabling Hsp70 to stably interact with client protein. This ATPase activation requires a transient, but highly specific, interaction between the J-protein’s J-domain and Hsp70. The evolutionary invariant His, Pro, Asp (HPD) sequence motif of the J-domain is critical for ATPase activation in all J-protein cochaperones tested (Mayer and Bukau 2005; Kampinga and Craig 2010).

**Comment by ValWood** [comment]
>I think that is likely what the obsoleted yesterday term was for! (grouping the carrier type ATPase with the TM atpase)!

not that I don't think we  need a grouping term  for the T-M transporters with the carriers in MF  (other than ATPase)

**Comment by Antonialock** [comment]
> But the activity is ATP-dependent ? Sounds like the mechanism is different from that of NfuA of E. coli ? See PMID:22966982
I think NfuA is different, I think it is a "protein maturation by iron-sulfur cluster transfer" protein

The bacterial protein that does "ATPase activity, coupled to translocation of iron-sulfur clusters " I think is hscA

![image](https://user-images.githubusercontent.com/12727506/93574360-3bacab00-f990-11ea-9357-4ce840f5b90c.png)


Fig. 3. The interplay between the scaffold and the chaperone within the ISC system. All components involved in the assembly and the release steps are predicted to arise in different conformations. IscU occurs either in a disordered (D) or structured (S) state. In its D state, IscU interacts with the IscS dimer. Conversion of IscU to the S state is accompanied by Fe/S cluster formation. IscU under its S state has little affinity for IscS, and HscB can displace it easily. HscB targets the ATP-bound HscA chaperone, which is in a tense state (T), to holoIscU leading to a temporary HscA/ATP/HscB/holoIscU complex. The substrate binding domain (SBD) of HscA binds the conserved sequence motif 99LPPVK103 of IscU. Then, the apoform of an acceptor protein (apoT) could attack the Fe/S cluster of IscU, leading to ATP hydrolysis and conversion of S state to D state of IscU. Thus, the ADP-bound form of HscA could stabilize the D state of IscU, which releases the Fe/S cluster. Then, exchange of ADP with ATP leads to a conformational change of HscA, which releases apoIscU under its D state, ready to engage in a new cycle of interaction with IscS.

**Comment by ValWood** [comment]
It might be useful to ask an expert here....

**Comment by ValWood** [comment]
related
https://github.com/geneontology/go-annotation/issues/4134#event-6473594230

it seems like:
1. BP GO:0097428 protein maturation by iron-sulfur cluster transfer 
https://github.com/geneontology/go-annotation/issues/4134#event-6473594230
should be replaced by a molecular function term ?

2. Not all "iron-sulfur cluster transfer "activities are ATPase dependent.

**Comment by ValWood** [comment]
Do we need this term?

It seems that pombe ssc1 is not precisely annotated 

<img width="1265" alt="Image" src="https://github.com/user-attachments/assets/bcf97b1c-609b-4777-b29a-59e175c43c0d" />

and the IEA is incorrect.

The term @Antonialock  suggested seems to make sense

**Comment by ValWood** [comment]
Is this the term we need?
https://www.ebi.ac.uk/QuickGO/term/GO:0140663
ATP-dependent FeS chaperone activity
Binding to and delivering metal ions to a target protein, driven by ATP hydrolysis. PMID:16843540

**Comment by ValWood** [comment]
Note to self, if [GO:0140663](https://www.ebi.ac.uk/QuickGO/term/GO:0140663)
is the correct term, update https://www.pombase.org/gene/SPAC664.11
and get Interpro mapping fixed

**Comment by pgaudet** [comment]
This seems to be the correct term.

**Comment by ValWood** [comment]
I  opened a ticket for the InterPro family
https://github.com/geneontology/go-annotation/issues/5748

@PCarme could you update ssc1
tks

**Comment by ValWood** [comment]
can close then?

**Comment by PCarme** [comment]
> I opened a ticket for the InterPro family [geneontology/go-annotation#5748](https://github.com/geneontology/go-annotation/issues/5748)
> 
> [@PCarme](https://github.com/PCarme) could you update ssc1 tks

As @Antonialock mentioned higher in the issue, this chaperone doesn't seem to bind the Fe-S cluster directly, so this term might not fit.

**Comment by ValWood** [comment]
Back to the drawing board. Although maybe this is an issue with the term definition.

@Antonialock requested
GO:NEW ATPase activity, coupled to translocation of iron-sulfur clusters
def: Catalysis of the reaction: ATP + H2O = ADP + phosphate, to directly drive the translocation of iron-sulfur clusters from one protein to another

we have
GO:0140663 ATP-dependent FeS chaperone activity
Binding to and delivering metal ions to a target protein, driven by ATP hydrolysis. PMID:16843540

 PMID:16843540
Mechanisms of iron–sulfur protein maturation in mitochondria,
cytosol and nucleus of eukaryotes

so this is likely what it was intended for?

GO:0140663 ATP-dependent FeS chaperone activity
is used for
IPR019591
but does not appear to have any EXP annotations

**Comment by ValWood** [comment]
ok IPR019591  is nbp35 https://www.pombase.org/gene/SPAC637.08

**Comment by ValWood** [comment]
So, Human HSPA9(Hsp70) (S.p/S.c Ssc1) , transfers the iron sulfur cluster from H.s. IscU  (S.p/S.c isu1) to H.s GLRX5(Grx5) Sp/Sc grx5)
using ATP, but does not itself interact with the Fe-S cluster? 

I don't know how we describe that as a MF.....

**Comment by pgaudet** [comment]
How does it transfer it without interacting with it ? you mean this should be a BP ? or is the complex that mediates the activity?

**Comment by deustp01** [comment]
> but does not itself interact with the Fe-S cluster?

The PubMed entry for human HSPA9 cites a paper PubMed:[26702583](https://www.uniprot.org/citations/26702583) whose abstratc recites genetic and indirect evidence that the protein is required for successful iron sulfur cluster transfer. That looks more like absence of evidence one way or the other about direct interaction, not evidence against interaction.

**Comment by ValWood** [comment]
Maybe it changes the conformation of the protein it acts in  without interacting directly with the iron sulfur cluster?

**Comment by ValWood** [comment]
It's

<img width="121" alt="Image" src="https://github.com/user-attachments/assets/2be4efd2-57f3-4b28-8dff-c2df97ec922d" />

 this part of the figure above. Hsc70 aka HSPAP aka Ssc1 is the only thing with ATP as so it must  be the responsible ATPase for the iron sulfur cluster transfer,

**Comment by ValWood** [comment]
this is clearer
https://user-images.githubusercontent.com/12727506/46525078-940d3f80-c882-11e8-8082-63dbbc28ff6c.png
